### PR TITLE
New: Native Pie chart

### PR DESCRIPTION
**Pull request by abhijit945**
**What is the purpose of this pull request? (put an "X" next to item)**

[] Documentation update
[] Bug fix
[X] New functionality 
[] Changes an existing functionality
[] Other, please explain:

**What changes did you make? (Give an overview)**
Created Native Pie Chart

Todo:
- [x] Documentation
- [x] Legend Label Formatter

**Add any screenshots**
image

image

image

image

### CODE REVIEW

**Comment by abhijit945** [comment]
@ryanthemanuel Can you add some members of your team in this review? I'll be happy to answer any questions.

**Comment by abhijit945** [comment]
@Dinesh94Singh

**Comment by Dinesh94Singh** [comment]
+1

File: src/main/js/controls/Pie/PieContent.js [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+        label: dataTarget.label || {},
+        key: dataTarget.key
+    });
+/**
+ * Validates the newly added pie content into the graph before rendering
+ * @private
+ * @param {Object} input - Newly added graph tasks
+ * @throws {module:errors.THROW_MSG_NO_DATA_LOADED}
+ * @throws {module:errors.THROW_MSG_NO_DATA_POINTS}
+ * @throws {module:errors.THROW_MSG_INVALID_DATA_PROPERTY}
```
Comment (Dinesh94Singh) [inline src/main/js/controls/Pie/PieContent.js]: add throws for Label not provided as well

**Pull request merged**